BUILD.md

```diff
@@ -57,3 +58,9 @@ You can use it by replacing the step 3 with the following:
  - If step 4 gives you an error that contains something like `main.cpp.o: in function 'std::__cxx11::basic_string...`, you likely are experiencing a clang-14 issue. This can only be fixed by either lowering the clang version or using GCC, see below.
  - If step 4 gives you a different error, you could report it to this repo or try using GCC. Just make sure your standard library and compilers are updated since Cemu uses a lot of modern features!
 - If step 4 gives you undefined libdecor_xx, you are likely experiencing an issue with sdl2 package that comes with vcpkg. Delete sdl2 from vcpkg.json in source file and recompile
+
+## Updating Cemu version
```

Comment:
```suggestion
## Updating Cemu And Source Code
```
Just a tad clearer that you aren't just updating Cemu but also the source code, since this guide is targeting mostly developers, at least in the future. The prebuild packages that come from cemu.info handle the updating on their own, probably.